### PR TITLE
Use NVPL for BLAS in CUDA SBSA Wheel 

### PR DESCRIPTION
**Pull request by tinglvv**
Use NVPL to replace OpenBLAS in CUDA SBSA Wheel for better cpu performance 

cc @ptrblck @Aidyn-A @nWEIdia @atalman

### CODE REVIEW

File: aarch64_linux/aarch64_wheel_ci_build.py [hunk 1]
```diff
@@ -86,7 +86,6 @@ def update_wheel(wheel_path) -> None:
         "/usr/local/cuda/lib64/libcudnn_engines_precompiled.so.9",
         "/usr/local/cuda/lib64/libcudnn_heuristic.so.9",
         "/opt/conda/envs/aarch64_env/lib/libgomp.so.1",
-        "/opt/OpenBLAS/lib/libopenblas.so.0",
```
Comment (Aidyn-A) [inline aarch64_linux/aarch64_wheel_ci_build.py:89]: Where do you copy `libnvpl_*`?

File: manywheel/Dockerfile_cuda_aarch64 [hunk 2]
```diff
@@ -74,17 +74,20 @@ ARG BASE_CUDA_VERSION
...
 FROM final as cuda_final
 ARG BASE_CUDA_VERSION
 RUN rm -rf /usr/local/cuda-${BASE_CUDA_VERSION}
 COPY --from=cuda     /usr/local/cuda-${BASE_CUDA_VERSION}  /usr/local/cuda-${BASE_CUDA_VERSION}
 COPY --from=magma    /usr/local/cuda-${BASE_CUDA_VERSION}  /usr/local/cuda-${BASE_CUDA_VERSION}
-COPY --from=openblas     /opt/OpenBLAS/  /opt/OpenBLAS/
+COPY --from=nvpl     /usr/local/nvpl-lib/  /usr/local/nvpl-lib/
+COPY --from=nvpl     /usr/local/nvpl-include/  /usr/local/nvpl-include/
+RUN cp -nr /usr/local/nvpl-lib/* /usr/local/lib/
+RUN cp -nr /usr/local/nvpl-include/* /usr/local/include/
```
Comment (Aidyn-A) [inline manywheel/Dockerfile_cuda_aarch64:90]: Assuming you agree installing it to `/opt/nvpl`:

```suggestion
COPY --from=nvpl /opt/nvpl/lib/  /usr/local/lib/
COPY --from=nvpl /opt/nvpl/include/  /usr/local/include/
```

File: aarch64_linux/aarch64_wheel_ci_build.py (hunk 1 above)
Comment (tinglvv) [reply aarch64_linux/aarch64_wheel_ci_build.py:89]: Thanks for noticing! Let me add those 😄

**Comment by tinglvv** [comment]
closing to move to https://github.com/pytorch/pytorch/pull/132811

**Pull request closed without merging**